dotnet/samples/GettingStartedWithProcesses/Step04/Steps/ManagerAgentStep.cs

```diff
@@ -3,6 +3,8 @@
 using System.ComponentModel.DataAnnotations;
```

Comment:
do all the bot warnings get fix with removing this import? `System.ComponentModel.DataAnnotations;`

Reply:
Yes, I believe so.  I thought I had already removed it, but must have not commited.  Been distracted on the investigating the alternate approach.  Updating namespace now to stabilize this version.